src/components/Property/Types/VectorProperty/ViewOptions/ColorView.tsx

```diff
@@ -20,38 +39,71 @@ export function ColorView({
   additionalData,
   isInt
 }: Props) {
-  if ((value.length !== 3 && value.length !== 4) || isInt) {
-    throw Error('Invalid use of Color view option!');
-  }
+  const [isError, setIsError] = useState(false);
+
+  const isOutsideRange = value.some((v) => v < 0 || v > 1);
+  const currentColor: RGBA = openspaceColorToRgba(value);
   const hasAlpha = value.length === 4;
+  const valueDisplayString = value
+    .map((v) => parseFloat(v.toFixed(3)).toString())
+    .join(', ');
+
+  validateInput(value, additionalData, isInt);
+
+  function isValidColorString(colorString: string): boolean {
+    const newValues = colorString.split(',').map((v) => parseFloat(v));
+    const result =
+      newValues.length === value.length &&
+      newValues.every((v) => !isNaN(v) && isNumber(v));
+
+    return result;
```

Comment:
```suggestion
    return (
      newValues.length === value.length &&
      newValues.every((v) => !isNaN(v) && isNumber(v))
    );
```
`result` is redundant here and to see what the function returns one has to go up to the variable anyways